fixposition_driver_ros1/msg/Speed.msg

```diff
@@ -13,9 +13,8 @@
 #
 ####################################################################################################
 
```

Comment:
Can/Should we support user timestamping?

Reply:
That is the plan in the semi near future, and the fpb measurements has the infrastructure for it. Currently however we dont support it internally.